README.md

```diff
@@ -1,6 +1,26 @@
-# Copenhagen Theme by Zendesk
+# MaxMind ZenDesk Theme
 
-The Copenhagen theme is the default Zendesk Guide theme. It is designed to be responsive and accessible.
+The MaxMind ZenDesk theme is a fork of [the Copenhagen theme](https://github.com/zendesk/copenhagen_theme) (the default Zendesk Guide theme).
+
+## Setting Up the Development Environment
+Dependencies and tools for previewing changes locally have been implemented. In order to use this remote development environment so that you have the proper versions of various dependencies without worrying about conflicts on your local machine, proceed with the following steps:
```

Comment:
I'd remove `remote` since this is actually on a local docker container

Reply:
removed